concent_api/middleman/middleman_server.py

```diff
@@ -0,0 +1,83 @@
+import asyncio
+
+import socket
+import sys
+from logging import getLogger
+
+from middleman.decorators import safe_execution
+
+PORT = 8888
+
+
+logger = getLogger(__name__)
+
+
+class MiddleMan(object):
+    def __init__(self, host=None, port=None):
+        self.host = host or socket.gethostname()
+        self.port = port or PORT
+        self.loop = None
+        self.counter = 0
```

Comment:
Shouldn't there be some kind of a global connection limit ?

Reply:
None I am aware of ;) At this point I do not think it is necessary. @cameel ?

Reply:
`syncio.start_server()` does have a limit (deafault = `5` AFAIK)